examples/transloadit/README.md

```diff
@@ -0,0 +1,24 @@
+# Robodog
+
+This example shows all the different Robodog APIs in action on a single page.
```

Comment:
This first sentence doesn't make sense here anymore
```suggestion
```

Reply:
I mean it still shows all the different Robodog APIs on a single page, I think we should keep it.

Reply:
But it doesn't show different **Robodog APIs**, it shows **alternatives**. I think we either include that word or remove the sentence :)

Reply:
For folks following at home, this was discussed during the team meeting today, we agreed to keep this as a migration example, and therefor keep the Robodog references as it is aimed solely to Robodog users looking for a migration path out of Robodog now that we have deprecated it and plan on remove it on a followup PR.